src/components/Loading.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useContext, useEffect } from 'react';
+import styled from 'styled-components';
+import { LoadingContext } from 'components/App';
+import DELAY_MS from 'constants/delay';
+
+function Loading() {
+  const { setLoading } = useContext(LoadingContext);
+  useEffect(handleLoading, []);
```

Comment:
```suggestion
  useEffect(function handleLoading() {
    setTimeout(() => setLoading(false), DELAY_MS.LOADING);
  }, []);
```

취향일수도 있는 부분인데, 저는 이런 형태가 코드(콜백)가 하는 일을 아래 라인(L16)으로 갈 필요 없이 바로 확인할 수 있어서 가독성에 더 좋다고 생각해요.